toolchain/binutils/patches/2.40/039-LoongArch-ld-Fix-relocation-error-of-pcrel.patch

```diff
@@ -0,0 +1,128 @@
+From 509a2ec6ad3ea7eb3f4cf59538cf636a2126e4c3 Mon Sep 17 00:00:00 2001
+From: liuzhensong <liuzhensong@loongson.cn>
+Date: Fri, 2 Sep 2022 16:29:14 +0800
+Subject: [PATCH 039/160] LoongArch:ld: Fix relocation error of pcrel.
```

Comment:
This is a backport from the 2.39 stable branch. It should be included in binutils 3.40.
I am not sure if it was really added to the 2.40 branch, but I think you should remove this patch.

Reply:
This patch didn’t add to the 2.40 branch. But I will remove this.

Reply:
It is strange that this patch is not in the 2.40 branch.
Probably there is a reason they only added it to the 2.39 branch. Maybe they have a better fix in 2.40.